src/uninstall_arg.rs

```diff
@@ -0,0 +1,68 @@
+use std::process::Command;
+
+/// Uninstaller Argument
+#[derive(Debug)]
+pub struct UninstallArg {
```

Comment:
Suggest have different types of builder.  For example, `helm.install_builder()`, `helm.uninstall_builder()`. etc